modules/core/src/test/java/org/apache/ignite/cdc/AbstractCdcTest.java

```diff
@@ -222,37 +224,32 @@ protected GridAbsPredicate sizePredicate(
 
     /** */
     protected void checkMetrics(CdcMain cdc, int expCnt) throws Exception {
-        IgniteConfiguration cfg = getFieldValue(cdc, "igniteCfg");
```

Comment:
Why do you need this changes?

Reply:
It's redundant code, that I left by mistake from previous commits. Fixed it